client/src/snippets.json

```diff
@@ -0,0 +1,23 @@
+{
+    "Basic Galaxy tool": {
+        "prefix": "gtool",
```

Comment:
I see this is too easy :)
But I guess we should have a common prefix?

Would `gx-` work?

Reply:
Yes ``gx-`` could be a good starting prefix for all snippets. I'll update the sample to ``gx-tool``